ubyssey/events/management/commands/EventRSS.py

```diff
@@ -0,0 +1,113 @@
+import re, datetime
+
+import feedparser
+
+from django.core.management.base import BaseCommand
+
+from ubyssey.events.models import Event, ScrapedEvent
+
+class RSSFeed(object):
```

Comment:
I'd name this `UBCEventsRSSFeed` to avoid confusion later

Reply:
Also, it might be nicer to name the file something like `import_events` so that the command is:

```
python manage.py import_events
```